layouts/index.html

```diff
@@ -85,8 +90,14 @@ <h2 class="title--answer">Mostly...</h2>
       </li>
       <li class="list__item--ok">
         Output/display configuration tool:
+        <a href="https://gitlab.com/kwinft/disman">Disman</a>,
         <a href="https://github.com/emersion/kanshi">kanshi</a>,
+        <a href="https://invent.kde.org/plasma/kscreen">KScreen</a>,
+        <a href="https://invent.kde.org/plasma/libkscreen">kscreen-doctor</a>,
```

Comment:
I would remove this since it is a dev lib.

Reply:
I'd agree to remove KScreen because of redundancy and because it comes by default, but kscreen-doctor is pretty much an xrandr clone for Plasma, like wlr-randr is for wlroots.